### PR TITLE
Add addE and deleteE commands to User Guide

### LINKED ISSUE #78: Update User Guide with addE and deleteE commands

### PR DESCRIPTION
**Pull request by aslam341**
Fix #78

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -72,19 +72,27 @@ Shows a message explaning how to access the help page.
...
+* Email: john.doe@gmail.com 
+* Address: Any string resembling a location
+* Tag (optional): Any string
+
+
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `adde n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
+* `adde n/Betsy Crowe t/Junior SWE e/betsycrowe@example.com a/Brick street p/+65 9123 4567 t/Employee of the month`
```
Comment (rayyan35p) [inline docs/UserGuide.md:95]: Are the 'e's in `adde` supposed to be capitalised?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -72,19 +72,27 @@ Shows a message explaning how to access the help page.
...
 
 
-### Adding a person: `add`
+### Adding an employee: `addE`
 
-Adds a person to the address book.
+Adds an employee to the employees list.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `adde n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
```
Comment (rayyan35p) [inline docs/UserGuide.md:79]: 'e' in `adde` not capitalised. Perhaps it should be based on what I see of the other formats you have written below.

**Review by rayyan35p — CHANGES_REQUESTED** [review]
Just one minor issue, other than that it looks good.

File: docs/UserGuide.md (hunk 1 above)
Comment (aslam341) [reply docs/UserGuide.md:95]: Yes, thanks for spotting that, it is supposed to be `addE`.

File: docs/UserGuide.md (hunk 2 above)
Comment (aslam341) [reply docs/UserGuide.md:79]: Good catch, this is also supposed to be `addE`.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -72,19 +72,27 @@ Shows a message explaning how to access the help page.
...
-A person can have any number of tags (including 0)
+An employee can have any number of tags (including 0)
 </div>
 
+Acceptable values for each parameter:
+* Name: John Doe / Alex / A. P. Mary
+* Phone number: +65 9123 4567 / 62353535 
+* Email: john.doe@gmail.com 
+* Address: Any string resembling a location
+* Tag (optional): Any string
```
Comment (antonTan96) [inline docs/UserGuide.md:90]: We do not have the phrase `Acceptable values for each parameter` in findE and addP. Maybe we could refactor this part to be expressed in sentences for consistency.

**Review by antonTan96 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (aslam341) [reply docs/UserGuide.md:90]: I'll remove it then, since other commands do not have this.

**Review by antonTan96 — APPROVED** [review]
LGTM!

**Pull request merged**